src/main/java/org/opentripplanner/model/StreetNoteAndMatcher.java

```diff
@@ -0,0 +1,52 @@
+package org.opentripplanner.model;
+
+import java.io.Serializable;
+import java.util.Objects;
+import org.opentripplanner.framework.tostring.ToStringBuilder;
+
+/**
+ * A container for a pair (note and note matcher).
+ *
+ * @author laurent
+ */
+@SuppressWarnings("ClassCanBeRecord")
```

Comment:
Is there a reason why to add this supression?

Reply:
IntelliJ want to convert into into a record, but it should not:
 - It is a model class - should have behaviour 
 - toString
 - ref none record classes
 - JavaDoc
 - possible encapsulation problems